spec/controllers/pageflow/editor/file_import_controller_spec.rb

```diff
@@ -152,12 +210,18 @@ module Pageflow
 
       it 'starts the file import job for each selected file' do
         user = create(:user)
-        entry = create(:entry, with_editor: user)
+        file_importer = FileImporterHelper.new
+        entry = create(:entry,
+                       with_editor: user,
+                       features_configuration: {file_importer.feature_name => true})
         sign_in(user, scope: :user)
-        file_importer = FileImporterDummy.new
+
         pageflow_configure do |config|
-          config.file_importers.register(file_importer.name, file_importer)
+          config.features.register(file_importer.feature_name) do |feature_config|
+            feature_config.file_importers.register(file_importer)
+          end
         end
+
         selected_files = file_importer.search(nil, nil)['photos']
         meta_data = file_importer.files_meta_data(nil, selected_files)
         post(:start_import_job, params: {files: meta_data['files'],
```

Comment:
Especially for the action that starts the job and performs the actual import, I'd say it's important to also test that it is not possible if the feature is disabled. This only tests the happy path.